cms/djangoapps/contentstore/views/course.py

```diff
@@ -464,10 +464,16 @@ def _accessible_libraries_iter(user, org=None):
     """
     List all libraries available to the logged in user by iterating through all libraries.
 
-    If 'org' is present, only libraries from that org will be returned.
+    org (string): if not None, this value will limit the libraries returned. An empty
+        string will result in no libraries, and otherwise only libraries with the
+        specified org will be returned. The default value is None.
     """
+    if org is not None:
+        libraries = [] if org == '' else modulestore().get_libraries(org=org)
```

Comment:
`org` had been defaulting to '', [see here](https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/course.py#L482)

`modulestore().get_libraries(org=org)` will happily accept an empty string for 'org', and return all unfiltered libraries.